klangbecken.py

```diff
@@ -1114,6 +1346,11 @@ def main(dev_mode=False):
             Specify alternate port [default: 5000].
       -b ADDRESS, --bind=ADDRESS
             Specify alternate bind address [default: localhost].
+      -s PLAYER_SOCKET, --socket=PLAYER_SOCKET
+            Set the location or address of the liquisoap player socket.
+            This can either be the path to a UNIX domain socket file or
+            a domain name and port seperated by a colon (e.g. localhost:123)
+            [default: ./klangbecken.sock]
```

Comment:
I *think* this contradicts https://github.com/radiorabe/klangbecken/blob/0c753bafc14a3caea7dcc71a35491ed85ec60a30/klangbecken.py#L678

Reply:
This _kind_ of contradicts. Actually, the docopt magic makes `./klangbecken.sock` the default value.

But the function default argument is completely useless, as it's getting overwritten. I'll remove this default argument and all the other useless ones from the function signatures.